Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator/Jobs/SendSearchRequestToDrivers/Handle/Internal/DriverPool.hs

```diff
@@ -108,6 +112,8 @@ prepareDriverPoolBatch driverPoolCfg searchReq searchTry batchNum = withLogTag (
           else calculatePool batchSize sortingType onlyNewDrivers allNearbyDrivers intelligentPoolConfig transporterConfig
       pure $ addDistanceSplitConfigBasedDelaysForDriversWithinBatch driverPoolWithoutBatchSplitDelays
       where
+        getBlockListedDrivers = Redis.getList
```

Comment:
why this?

Reply:
Alex told this to make it as a separate function

Reply:
> Alex told this to make it as a separate function

I meant something like `getBlockListedDrivers = Redis.getList . mkBlockListedDriversKey`. Currently it's indeed doesn't make sense

Reply:
Truee